### PR TITLE
chore(coinmarket): coinmarket cleaning code

### LINKED ISSUE #14703: Coinmarket - cleaning the code
## Description 
- delete unused files & types, update export of types
- rename folders `sell_new` to `sell` and `exchange_new` to `exchange`
- use `export` instead of `export default`
- use absolute paths instead of relative paths
- use upper snake case for constants
- delete unused translations
- delete unused CSS styles
- update test crypto symbols to CryptoId
- replace index files with name of component

### PR DESCRIPTION
**Pull request by adderpositive**
## Description 
- delete unused files & types, update export of types
- rename folders `sell_new` to `sell` and `exchange_new` to `exchange`
- use `export` instead of `export default`
- use absolute paths instead of relative paths
- use upper snake case for constants
- delete unused translations
- delete unused CSS styles
- update test crypto symbols to CryptoId
- replace index files with name of component

## Related
Resolved https://github.com/trezor/trezor-suite/issues/14703

### CODE REVIEW

**Review by MiroslavProchazka — APPROVED** [review]

File: packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketOfferExchange/CoinmarketOfferExchangeSendSwap.tsx [hunk 1]
```diff
@@ -139,11 +139,7 @@ const slippageOptions = [
     },
 ];
 
-export function formatCryptoAmountAsAmount(
-    amount: number,
-    baseAmount: number,
-    decimals = 8,
-): string {
+function formatCryptoAmountAsAmount(amount: number, baseAmount: number, decimals = 8): string {
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketOfferExchange/CoinmarketOfferExchangeSendSwap.tsx]: const

File: packages/suite/src/views/wallet/coinmarket/index.tsx [hunk 2]
```diff
@@ -35,6 +27,7 @@ export const CoinmarketWrapper = `
     }
 `;
 
+// reason: responsive sizes
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/coinmarket/index.tsx]: How about using Card Wrapper here instead of modifying the card itself?

**Comment by adderpositive** [comment]
@tomasklim I have also added the renaming of the folders. It is in this commit - https://github.com/trezor/trezor-suite/pull/14728/commits/cc5456d93e0ff2a422212585e6f71ff6154d1b9b

**Review by tomasklim — APPROVED** [review]

**Pull request merged**